.github/workflows/publish-package-docs.yaml

```
@@ -60,11 +60,12 @@ jobs:
           python -m pip install setuptools six wheel
 
       - name: Install project requirements
```

Shouldn't this be:
```suggestion
      - name: Install ansible-core requirements
```
?